.changeset/tame-feet-applaud.md

```diff
@@ -0,0 +1,11 @@
+---
+'@twilio-paste/styling-library': minor
+'@twilio-paste/core': minor
```

Comment:
question: as this is a major to Emotion, should we follow suit and make this a major version change to Paste? 

One the face of it, it looks like nothing really changed from a consumer API, so I don't think you'll experience a breaking change, but should we air on the side of caution and all it out?